src/Http/AbstractHttpMessage.php

```diff
@@ -51,11 +53,12 @@ public function addHeader(string $name, string $value)
 
     /**
      * @param string $header
+     * @param mixed  $default
      * @return string
      */
-    public function getHeader($header)
+    public function getHeader(string $header, $default = null)
```

Comment:
@valanz you mean type hint of default should be string instead of mixed ?
@folliked can't typehint the return because it's compatible 7.0 and may return null